### ISSUE TITLE
RunTest task fails when is run over a SFDC org with namespace

### ISSUE THREAD

**Issue by J0c4** [fundacionjala/enforce-gradle-plugin#272]
Hi all

I found a problem when I try to run `gradle runTest` over a SFDC org with namespace prefix enabled. I reproduced this issue only with versions 1.1.3 and 1.1.4, previous versions don't have this problem, but I need the last ones because they have the support to upload Aura components. 

Actually I reviewed the code and the problem is in the line 148 of RunTestTask.groovy file. Based on [this doc](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/sforce_api_calls_runtests_request.htm), 'namespace' property shouldn't be used if allTests property is set to 'true'.

I receive the following error:
![image](https://cloud.githubusercontent.com/assets/4606607/15193043/01c122a0-178a-11e6-838b-3d648acf625e.png)

I've already forked the repo, but I didn't have chance to check how to validate my changes. I'd like to submit a PR to master branch with this fix but I don't know if you guys are planning to release a new version. If so, it'd be great to have this fix, please let me know what's the best approach to add it.

Thanks!

**Comment by ziur** [comment]
@J0c4, please submit your PR and we can generate a new release with your fix.

**Comment by J0c4** [comment]
@ziur  Thanks!. I have my commit, but I'm having troubles trying to create the PR. I receive error 500, any clue how can I fix it?

**Comment by JimmyRojas** [comment]
Hello @J0c4, it seems to be a github error, but meantime, what we can do is apply your fix directly thus we can validate it

Please confirm if those are all your changes

![diff_j0c4](https://cloud.githubusercontent.com/assets/13870974/15269520/12743f32-19cf-11e6-88a9-0742af321156.png)

**Comment by J0c4** [comment]
@JimmyRojas Yes, that's the fix. I'll keep trying to see if I can continue collaborating with the project. Thank you!
